build.py

```diff
@@ -75,8 +75,8 @@
     "triton_container_version": "24.12",
     "upstream_container_version": "24.12",
     "ort_version": "1.20.1",
-    "ort_openvino_version": "2024.5.0",
-    "standalone_openvino_version": "2024.5.0",
+    "ort_openvino_version": "2024.4.0",
```

Comment:
Why we downgrade version here?

Reply:
I didn't notice this file was changed in the commit "Update OpenVINO to 2024.5 (#7857)". Should be fixed now.